.github/ALLOWLIST

```diff
@@ -17,6 +17,7 @@ cmd/configschema
 examples/demo/client
 examples/demo/server
 exporter/influxdbexporter
+exporter/logicmonitorexporter
```

Comment:
New components must not be added to the allowlist: they must have an owner

Reply:
Removed the entry from allowlist. Thanks!

Reply:
Hi @mx-psi ,
After I removed the entry from allowlist, the check 'check-codeowners' is failing stating
> there are 1 components not included in CODEOWNERS and not known in the ALLOWLIST

Can you please guide me what should be done as no code owner has been assigned to PR yet.

Reply:
@khyatigandhi6 Hey, as documented [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#adding-new-components), you need to
> Add the sponsor for your component and yourself to a new line for your component in the [.github/CODEOWNERS](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS) file.

If you are contributing a new component, you are expected to own it and respond when pinged on issues or to review PRs related to this component.

Reply:
Thanks! Made changes accordingly.